src/common/hostPlatform.ts

```diff
@@ -53,6 +55,11 @@ class WindowsHostPlatform implements IHostPlatform {
     public getPlatformId(): HostPlatformId {
         return HostPlatformId.WINDOWS;
     }
+
+    public getTargetPlatformCompatibility(targetPlatformName: string): boolean {
```

Comment:
@dlebu, @joshuaskelton, @hamiltonia, what do you guys think about the method name?